lucille-core/src/main/java/com/kmwllc/lucille/util/SolrUtils.java

```diff
@@ -82,17 +79,19 @@ private static CloudSolrClient getCloudClient(Config config) {
    * @param config The configuration file to generate the HttpClient from.
    * @return the HttpClient
    */
-  public static HttpClient getHttpClient(Config config) {
-    HttpClientBuilder clientBuilder = HttpClientBuilder.create();
+  public static Http2SolrClient getHttpClient(Config config) {
+    Http2SolrClient.Builder clientBuilder = new Http2SolrClient.Builder();
 
     if (requiresAuth(config)) {
       CredentialsProvider provider = new BasicCredentialsProvider();
       String userName = config.getString("solr.userName");
       String password = config.getString("solr.password");
       UsernamePasswordCredentials credentials = new UsernamePasswordCredentials(userName, password);
       provider.setCredentials(AuthScope.ANY, credentials);
-      clientBuilder.setDefaultCredentialsProvider(provider);
-      clientBuilder.addInterceptorFirst(new PreemptiveAuthInterceptor());
+      clientBuilder.withBasicAuthCredentials(userName, password);
+      //clientBuilder.addInterceptorFirst(new PreemptiveAuthInterceptor());
+      //clientBuilder.withFollowRedirects(true);
+
     }
 
     return clientBuilder.build();
```

Comment:
```suggestion
    return clientBuilder.build();
    
    Need to investigate why this is causing an error "Couldn't initialize a HttpClusterStateProvider" and push those changes to prevent this
```